src/systems/battery_plugin/LinearBatteryPlugin.cc

```diff
@@ -108,8 +109,8 @@ class ignition::gazebo::systems::LinearBatteryPluginPrivate
   /// \brief Instantaneous battery charge in Ah.
   public: double q{0.0};
 
-  /// \brief State of charge [0, 1]
-  public: double soc{1.0};
+  /// \brief State of charge [0, 1].
+  public: double soc{100.0};
```

Comment:
Not sure why this was changed. It's causing a test failure. I think it makes sense for it to be 1.0.

Reply:
Totally uninteded. Reverted in f7fa498. I need a coffee or go to sleep ;)